### ISSUE TITLE
Element type is invalid: expected a string for built-in components or a class/function

### ISSUE THREAD

**Issue by annelorraineuy** [CRAlpha/react-native-wkwebview#34]
> Element type is invalid: expected a string for built-in components or a class/function but got: undefined. Check the render method of `WKWebview`

I'm using the RN 0.39 version on the branch `0.5.0-for-RN0.39`

This is how I loaded it: 
```
import WKWebView from 'react-native-wkwebview-reborn';

<WKWebView
                    ref="webview"
                    onProgress={(progress) => console.log(progress)}
                    sendCookies={true}
                    source={{
                        uri: gameUri,
                        headers: {
                            'Authorization': `${token_type} ${access_token}`,
                            'Accept': 'text/html; charset=utf-8',
                            'rooplay-installid': installId
                        },
                        method: 'get'
                    }}
                    automaticallyAdjustContentInsets={true}
                    style={webView}
                    bounces={false}
                    startInLoadingState={true}
                />
```

**Comment by insraq** [comment]
It seems that your WKWebView import fails? Try clean packager cache / rebuild project?

**Comment by annelorraineuy** [comment]
I did try both, refresh packager and rebuilt project.

**Comment by insraq** [comment]
Have you try to lock down which line reports error? Is your WKWebView undefined?

**Comment by okadan** [comment]
ActivityIndicatorIOS was removed in react-native >= 0.37

**Comment by okadan** [comment]
Fixed.

https://github.com/CRAlpha/react-native-wkwebview/pull/35/commits/ed32c2c30156ad9f28892cf10da10a216be3cc0b

**Comment by insraq** [comment]
@okadan That's interesting. We are on RN 0.39 and it seems fine. We will check after our team returns from holidays.

**Comment by insraq** [comment]
Fixed by v1.2.0

**Comment by annelorraineuy** [comment]
@okadan Yes, this is fixed! Sorry for the late response. Works perfectly now for me. Thank you so much!
